foreigndonors.php

```diff
@@ -125,23 +127,20 @@ function foreigndonors_civicrm_buildForm($formName, &$form) {
       $form->addRule('foreigndonor', ts('You must affirm you are not a prohibited donor as per State and Federal legislation'), 'required', NULL, 'client');
     }
     elseif ($domainId == 8 || $isNSW ) {
-      $form->assign('addNSWProhibitedDonor', TRUE);
       $label = "I am an Australian Citizen or Permanent Resident and am not a foreign donor";
       $form->add('Checkbox', 'foreigndonor', ts('I am an Australian Citizen or Permanent Resident and am not a foreign donor'));
       $form->addRule('foreigndonor', ts('You must affirm you are an Australian Citizen or Permanent Resident'), 'required', NULL, 'client');
       $form->add('Checkbox', 'prohibiteddonor', ts('I am not a prohibited donor as per NSW electoral legislation'));
       $form->addRule('prohibiteddonor', ts('You must affirm you are not a prohibited donor as per NSW electoral legislation'), 'required', NULL, 'client');
     }
     elseif ($domainId == 9 || $isACT ) {
-      $form->assign('addACTProhibitedDonor', TRUE);
       $label = "I am an Australian Citizen or Permanent Resident and am not a foreign donor";
       $form->add('Checkbox', 'foreigndonor', ts('I am an Australian Citizen or Permanent Resident and am not a foreign donor'));
       $form->addRule('foreigndonor', ts('You must affirm you are an Australian Citizen or Permanent Resident'), 'required', NULL, 'client');
       $form->add('Checkbox', 'prohibiteddonor', ts('I am not a prohibited donor as per ACT electoral legislation'));
       $form->addRule('prohibiteddonor', ts('You must affirm you are not a prohibited donor as per ACT electoral legislation'), 'required', NULL, 'client');
     }
     else {
-      $form->assign('addProhibitedDonor', FALSE);
```

Comment:
@andrew-cormick-dockery is this variable not used in the template?

Reply:
I can't see a reference anywhere. Maybe it used to be used but now is not?

Reply:
yeh ok makes sense then